$NetBSD: patch-ab,v 1.2 2006/06/13 13:24:36 dsainty Exp $

Make sure ioctl() gets a third dummy argument, for the three argument macro
defined in the NetBSD OSS emulation soundcard.h.

--- src/SoundOSS.c.orig	2002-01-16 04:46:43.000000000 +1300
+++ src/SoundOSS.c	2006-06-03 18:25:08.000000000 +1200
@@ -106,7 +106,7 @@
 		return;
 	}
 
-	if ( ioctl ( SoundFD, SNDCTL_DSP_RESET ) < 0 ) {
+	if ( ioctl ( SoundFD, SNDCTL_DSP_RESET, &dummy ) < 0 ) {
 		fprintf ( stderr, "Can't reset sound device\n" );
 		return;
 	}
